tendermint/src/version.rs

```diff
@@ -2,7 +2,7 @@ use serde::{Deserialize, Serialize};
 use std::fmt::{self, Display};
 
 /// Tendermint version
-#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, Default)]
```

Comment:
Should the default version really be empty string?

Reply:
Removed.